app/javascript/locales.js

```diff
@@ -0,0 +1,25 @@
+const locales = {
+  en: {
+    message: {
+      selector: {
+        button: 'Go to dashboard',
+      },
+    },
+  },
+  es: {
+    message: {
+      selector: {
+        button: 'Ir a dashboard',
+      },
+    },
+  },
+  ja: {
+    message: {
+      selector: {
+        button: 'こんにちは、世界',
+      },
+    },
+  },
+};
+
+export { locales };
```

Comment:
Prefer default export             import/prefer-default-export<br>Irregular whitespace not allowed  no-irregular-whitespace